editor/store/selectors.js

```diff
@@ -200,7 +211,7 @@ export function getCurrentPostLastRevisionId( state ) {
  * @returns {Object} Object of key value pairs comprising unsaved edits.
  */
 export function getPostEdits( state ) {
-	return state.editor.present.edits;
+	return get( state, [ 'editor', 'present', 'edits' ], {} );
```

Comment:
Just noting that you can use also string as 2nd param:

```js
return get( state, 'editor.present.edits', {} );
```
See 3rd example in [here](https://lodash.com/docs/4.17.4#get)

Reply:
@gziolo I know, this is a small performance consideration because lodash won't have to parse the string.

Reply:
Hehe, more keystrokes to save some CPU time. They should include `lodash` into JS engine by default and optimize that for us or teach Babel to do the transformation from string to array!

Reply:
That's a good catch about having `{}` as the default value. The same would apply to `[]`.

Reply:
One example where I worked around this was in `getBlockOrder`:

https://github.com/WordPress/gutenberg/blob/e2c824a618720fcb2164cda101ca5483e34066ce/editor/store/selectors.js#L33-L40

https://github.com/WordPress/gutenberg/blob/e2c824a618720fcb2164cda101ca5483e34066ce/editor/store/selectors.js#L788-L801

I'd not love if we start having many constants at the top of this file, but we also should try to avoid these fallbacks whenever possible (instead relying on the value reference living in state as the result of the reducer).

Reply:
There could be 2 constants, one for an empty array and one for an empty object. However I second your opinion that reducers should handle it.